bin/scripts/export-data/setting_limits_and_real_values.js

```diff
@@ -0,0 +1,356 @@
+                currentVal = currentEventLimit;
+
+                app_results['Event Keys'] = {"default": defaultVal, "set": currentVal, "real": realVal};
+
+                // SEGMENTS IN ONE EVENT
+                defaultVal = DEFAULTS.event_segment_limit;
+
+                let currentEventSegmentLimit = pluginsCollectionPlugins && pluginsCollectionPlugins.api && pluginsCollectionPlugins.api.event_segmentation_limit || undefined;
+                currentVal = currentEventSegmentLimit;
+
+                let eventSegments = eventsCollectionPerApp && eventsCollectionPerApp.segments || {};
+                realVal = Object.entries(eventSegments)
+                    .sort((a, b) => b[1].length - a[1].length)
+                    .reduce((acc, [key, value]) => {
+                        acc[key] = value.length;
+                        return acc;
+                    }, {});
+
+                app_results['Event Segments'] = {"default": defaultVal, "set": currentVal, "real": realVal};
+
+                // UNIQUE EVENT SEGMENT VALUES FOR 1 SEGMENT
+                defaultVal = DEFAULTS.event_segment_value_limit;
+
+                let currentEventSegmentValueLimit = pluginsCollectionPlugins && pluginsCollectionPlugins.api && pluginsCollectionPlugins.api.event_segmentation_value_limit || undefined;
+                currentVal = currentEventSegmentValueLimit;
+                realVal = {};
+                await Promise.all(realEvents.map(async(event) => {
+                    var shortEventName = common.fixEventKey(event);
+                    var eventCollectionName = "events" + crypto.createHash('sha1').update(shortEventName + app._id).digest('hex');
+
+                    try {
+                        var eventsSegmentsValues = await countlyDb.collection(eventCollectionName).aggregate([
+                            {
+                                "$match": {
+                                    "_id": { "$regex": "^no-segment_2024:0.*" }
+                                }
+                            },
+                            {
+                                "$addFields": {
+                                    "meta_v2": {
+                                        "$objectToArray": "$meta_v2"
+                                    }
+                                }
+                            },
+                            {
+                                "$project":
+                                {"meta_v2": {"$map": {"input": "$meta_v2", "in": {"k": "$$this.k", "v": {"$size": {"$objectToArray": "$$this.v"}}}}}}
+                            },
+                            {
+                                "$unwind": "$meta_v2"
+                            },
+                            {
+                                "$match": {
+                                    "meta_v2.k": { "$ne": "segments" }
+                                }
+                            },
```

Comment:
You should stop here.
Return result as array and convert as object in node.js side.
Reason for it - if there are a lot values we risk 16mb limit in single doc and also this formatting will be faster outside of mongodb